src/app/shared-component/status-drawer.component.ts

```diff
@@ -10,7 +10,7 @@ import { WorkItem } from './../work-item/work-item';
 @Component({
 	selector   : 'status-drawer',
 	templateUrl: './status-drawer.component.html',
-	styleUrls  : ['src/app/shared-component/status-drawer.component.css'],
+	styleUrls  : ['/status-drawer.component.css'],
```

Comment:
@sanbornsen May be your are using old loader. In the new code, they add "." if it doesn't exist. See this https://github.com/TheLarkInn/angular2-template-loader/blob/master/index.js#L8 In the same file, line no. 3, you can see that they handle white-space also.